Gems/PhysX/Code/Tests/Benchmarks/PhysXJointBenchmarks.cpp

```diff
@@ -184,17 +184,36 @@ namespace PhysX::Benchmarks
         : public PhysXBaseBenchmarkFixture
     {
     public:
```

Comment:
nit: remove 'public' visibility for the internal* functions (to match the others in this PR)